Use PortAudio not ALSA
Disable isahc default feature to avoid using curl/static-curl

Index: Cargo.toml
--- Cargo.toml.orig
+++ Cargo.toml
@@ -23,7 +23,7 @@ features = ["v2_60"]
 
 [dependencies.librespot]
 version = "0.4.2"
-features = ["alsa-backend", "pulseaudio-backend"]
+features = ["portaudio-backend"]
 
 [dependencies.protobuf]
 version = "2.25.2"
@@ -45,7 +45,8 @@ version = "^1.0.93"
 
 [dependencies.isahc]
 version = "^1.7.2"
-features = ["json"]
+default_features = false
+features = ["json", "http2", "text-decoding"]
 
 [dependencies.rand]
 version = "^0.8.5"
